### PR TITLE
02-Git: Add task and solution.

### PR DESCRIPTION
**Pull request by denanko**
Second pull request to correct branch and from home PC.

### CODE REVIEW

**Comment by alexposukhov** [comment]
Please remove (From home PC) from commit title. Add sign to commit description. If your request is ready for review set Label "Ready for review"

File: 02-Git/theGame.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+function playAgain {
+  echo "Would you like to play again? (y/n): "
+  read play
+  if [[ $play == "y" ]]; then
+    main
+  fi
+}
+
+function main {
+  player1
```
Comment (alexposukhov) [inline 02-Git/theGame.sh:66]: Please add some intro for player , just like "You may use following controls : "r", "p", "s", select one of them"

**Review by alexposukhov — CHANGES_REQUESTED** [review]
Please fix comments in merge request

**Comment by denanko** [comment]
Fix commit message. Add sign. Add intro for players. Please, review updated commit.

**Comment by alexposukhov** [comment]
Please change patch format according to project requirement:
https://github.com/Kernel-GL-HRK/gl-kernel-training-2021/wiki
**Formatting patches**
Commit messages should include:
subject line (started from [tags]);
descriptive body, separated by empty line;
Signed-off-by line (use -s option to git commit)
If a new exercise, you provide a solution for, is based on another exercise or third-party code, you should commit a clean copy first (don't forget to mention its origin in the commit message) and then commit a patch with your changes for this exercise.
If original code requires cleaning up (e.g. to conform code standards or to remove unneeded functionality) - do it as a separate commit (prior to providing your changes).

**Comment by alexposukhov** [comment]
Please don't forget to change label in request to "Ready for review"

**Comment by alexposukhov** [comment]
Please change PR description as well

**Comment by denanko** [comment]
I have created another pull request where I tried to make a better structure. Therefore I close this one

**Pull request closed without merging**